requirements/base.txt

```diff
@@ -43,3 +43,7 @@ numpy>=1.22.2 # not directly required, pinned by Snyk to avoid a vulnerability
 
 # Query engine - PostgreSQL
 psycopg2==2.9.5
+
+# AI Assistant
+openai==0.27.5
```

Comment:
remove openai and langchain from base installation since these are optional addons to querybook